cmd/convox/templates/init/ruby/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM convox/ruby
+
+# copy only the files needed for bundle install
+# uncomment the vendor/cache line if you `bundle package` your gems
```

Comment:
Not sure yet. It doesn't seem to make much of a speed difference but it does drop some internet-dependency. Unfortunately it's hard to conditionally copy a directory if it exists.